src/features/form/containers/RepeatingGroupsEditContainer.tsx

```diff
@@ -38,19 +81,59 @@ export function RepeatingGroupsEditContainer({
   multiPageIndex,
   setMultiPageIndex,
   filteredIndexes,
-}: IRepeatingGroupsEditContainer): JSX.Element | null {
-  const language = useAppSelector((state) => state.language.language);
+  group,
+  row,
+  node,
+  language,
+}: IRepeatingGroupsEditContainer & {
+  group: HRepGroup;
+  row: HRepGroupRow;
+  node: LayoutNode<AnyItem>;
+  language: ILanguage;
+}): JSX.Element | null {
   const textResources = useAppSelector((state) => state.textResources.resources);
-  const node = useResolvedNode(id);
-  const group = node?.item;
-  if (!group || !node || group.type !== 'Group' || !('rows' in group)) {
-    return null;
-  }
-
-  const row = group.rows[editIndex];
   const textsForRow = row?.groupExpressions?.textResourceBindings;
   const editForRow = row?.groupExpressions?.edit;
   const editForGroup = group.edit;
+  const rowItems = row.items;
+
+  const gridRef = useRef<HTMLDivElement | null>(null);
+  const prevGroupRowItems = useRef<HRepGroupChild[] | undefined>(undefined);
+
+  useEffect((): void => {
+    if (!gridRef.current || JSON.stringify(prevGroupRowItems.current) === JSON.stringify(rowItems)) {
+      return;
+    }
+    prevGroupRowItems.current = rowItems;
+
+    const isFocusable = (element: FocusableHTMLElement): boolean => {
+      const tagName = element.tagName.toLowerCase();
+      const focusableElements = ['a', 'input', 'select', 'textarea', 'button'];
+
+      if (element.tabIndex < 0) {
+        return false;
+      }
+
+      const isAvailable =
+        element.type !== 'hidden' || !element.disabled || (element.type.toLowerCase() === 'a' && !!element.href);
+
+      return focusableElements.includes(tagName) && isAvailable;
+    };
+
+    const findFirstFocusableElement = (container: HTMLElement): FocusableHTMLElement | undefined => {
+      return Array.from(container.getElementsByTagName('*')).find(isFocusable) as FocusableHTMLElement;
+    };
```

Comment:
The linter want us to return the value immediately. :) 
```suggestion
    const findFirstFocusableElement = (container: HTMLElement): FocusableHTMLElement | undefined => Array.from(container.getElementsByTagName('*')).find(isFocusable) as FocusableHTMLElement;
```